site/documentation/content/admin-guide/hono-kafka-client-configuration.md

```diff
@@ -3,20 +3,24 @@ title = "Hono Kafka Client Configuration"
 weight = 342
 +++
 
-Protocol adapters can be configured to use Kafka for the messaging. The Kafka client used there can be configured with 
-operating system environment variables and/or Java system properties.
+Several Hono components can be configured to support Kafka as the messaging infrastructure. The Kafka client used
+for this purpose can be configured by means of operating system environment variables and/or Java system properties.
 
 {{% notice info %}}
 The support of Kafka as a messaging system is currently a preview and not yet ready for production.
 The implementation as well as its APIs may change with the next version.
 {{% /notice %}}
 
-## Configure for Kafka based Messaging
+## Configuring Tenants to use Kafka based Messaging
+
+Hono's components by default support using AQMP 1.0 based messaging infrastructure to transmit messages hence and forth
+between devices and applications. Hono also supports using Kafka as the messaging infrastructure either as a replacement
+for or as an alternative in addition to the AMQP 1.0 based infrastructure.
+
+All tenants configured in Hono use the AMQP 1.0 based infrastructure by default. A tenant can also be configured
+to use Kafka instead of AMQP 1.0 by means of adding the property `ext/messaging-type` with value `kafka` to the
+[tenant's configuration]({{< relref "/api/tenant#tenant-information-format" >}}) in the registry.
```

Comment:
(Typo: AQMP)

I think it would be good to point out at the beginning here already (instead of in the info box below), that this is only relevant if *both* messaging systems have been configured. 
In the usual case that only one messaging system is used, the reader can skip this section then.